tools/build_defs/scala/scala.bzl

```diff
@@ -26,7 +26,7 @@ load(
 def warning(rule):
   return """This rule has moved out of @bazel_tools!
 
-The scala rules have moved to https://github.com/bazelbuild/scala_rules, you
+The scala rules have moved to https://github.com/bazelbuild/rules_scala, you
 should now refer them via @io_bazel_rules_scala, use:
 
```

Comment:
Yes that's a typo

On Thu, Mar 3, 2016, 9:31 AM Philipp Wollermann <notifications@github.com>
wrote:

> In tools/build_defs/scala/scala.bzl
> <https://github.com/bazelbuild/bazel/pull/980#discussion_r54915496>:
>
> > @@ -26,7 +26,7 @@ load(
> >  def warning(rule):
> >    return """This rule has moved out of @bazel_tools!
> >
> > -The scala rules have moved to https://github.com/bazelbuild/scala_rules, you
> > +The scala rules have moved to https://github.com/bazelbuild/rules_scala, you
> >  should now refer them via @io_bazel_rules_scala, use:
> >
>
> @johnynek <https://github.com/johnynek> Mhm... I think so, too.
>
> @damienmg <https://github.com/damienmg> Can you have a look at
> https://github.com/bazelbuild/bazel/blob/master/tools/build_defs/scala/scala.bzl?
> I think you made those changes and probably know best what they're supposed
> to look like :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bazelbuild/bazel/pull/980/files#r54915496>.
>